cw-orc/src/daemon/queriers/bank.rs

```diff
@@ -47,4 +47,93 @@ impl Bank {
             }
         }
     }
+
+    /// Query spendable balance for address
+    pub async fn spendable_balances(
+        &self,
+        address: impl Into<String>,
+        pagination: Option<PageRequest>,
+    ) -> Result<cosmos_modules::bank::QuerySpendableBalancesResponse, DaemonError> {
+        let spendable_balances: cosmos_modules::bank::QuerySpendableBalancesResponse = cosmos_query!(
+            self,
+            bank,
+            spendable_balances,
+            QuerySpendableBalancesRequest {
+                address: address.into(),
+                pagination: pagination,
+            }
+        );
+        Ok(spendable_balances)
+    }
+
+    /// Query total supply in the bank
+    pub async fn total_supply(
+        &self,
+        pagination: Option<PageRequest>,
+    ) -> Result<cosmos_modules::bank::QueryTotalSupplyResponse, DaemonError> {
```

Comment:
Sometimes the Results that are returned from a query are simple wrappers around a value. In that case it's easier for the user of the querier that you already unwrap the value for them. 
Here for example you could return the total supply as just a number instead of returning the `QueryTotalSupplyResponse` type (which just wraps the number).

Reply:
ok, ill change the ones that are just wrapping basic data!

Reply:
Yeah I think that makes sense, altough it might be better to expose the pagination anyway. Can we add it as an optional parameter where it applies?